docs/install/index.md

```diff
@@ -229,7 +229,7 @@ $ sudo apt-get install -y libopencv-dev
 **Step 4** Download MXNet sources and build MXNet core shared library.
 
 ```bash
-$ git clone --recursive https://github.com/dmlc/mxnet
+$ git clone --recursive https://github.com/apache/incubator-mxnet
 $ cd mxnet
```

Comment:
Please also update `cd mxnet` to `cd incubator-mxnet `

Reply:
Done